packages/libllvm/llvm-config.in

```diff
@@ -40,68 +40,80 @@ Typical components:
   engine            Either a native JIT or a bitcode interpreter."
 }
 
-arch=@TERMUX_ARCH@
 version=@TERMUX_PKG_VERSION@
 prefix=@TERMUX_PREFIX@
 has_rtti=NO
 CPPFLAGS="-I${prefix}/include -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS"
-CFLAGS="${CPPFLAGS} ${CFLAGS}  -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS"
```

Comment:
I guess the second `CFLAGS` is to get those flags if they're set in the environment? It works for that, but I checked and the `llvm-config` binary doesn't do that. No idea why the `*_MACROS` flags were repeated here, considering they were already added by the preprended `CPPFLAGS`.